src/ray/object_manager/plasma/create_request_queue.cc

```diff
@@ -100,20 +100,24 @@ Status CreateRequestQueue::ProcessRequests() {
       }
 
       if (oom_start_time_ns_ == -1) {
+        // RAY_LOG(DEBUG) << "@lsf reset oom_start_time_ns_";
         oom_start_time_ns_ = now;
       }
       if (spill_objects_callback_()) {
+        // RAY_LOG(DEBUG) << "@lsf spill_objects_callback_()";
+        oom_start_time_ns_ = now;
```

Comment:
What do you think about resetting to -1 here? So this would only start the timer once there's on more spilling in progress.